package.json

```diff
@@ -9,9 +9,10 @@
     "build": "NODE_ENV=production yarn build:webpack",
     "build:webpack": "webpack",
     "build:watch": "webpack --watch",
-    "lint": "yarn lint:eslint && yarn lint:prettier",
+    "lint": "yarn lint:eslint && yarn lint:prettier && yarn lint:ec",
     "lint:eslint": "eslint .",
     "lint:prettier": "prettier --check resources/assets",
+    "lint:ec": "ec --exclude \"node_modules|vendor|public/assets|.git\"",
```

Comment:
`public/coverage` /for phpunit coverage reporting) and eventually public `*.xml` files should be excluded too (to be able to import mock fahrplan data from localhost).
Other than that it looks good and can be merged :heart_eyes:

Reply:
to be continued…